price-feeder/oracle/oracle.go

```diff
@@ -362,8 +384,7 @@ func GenerateSalt(length int) (string, error) {
 		return "", fmt.Errorf("failed to generate salt: zero length")
 	}
 
-	n := length / 2
-	bytes := make([]byte, n)
+	bytes := make([]byte, length)
```

Comment:
Btw, this is handled in https://github.com/umee-network/umee/blob/a63399da0d041282149b35ee577e72174579b436/x/oracle/types/msgs.go#L65 already 👍

Reply:
got it

Reply:
Looks like it still works though. So let's keep it.